daliuge-engine/test/apps/test_docker.py

```diff
@@ -158,13 +160,15 @@ def assertMsgIsCorrect(msg, command):
         msg = 'This is a message with a double quotes: "'
         assertMsgIsCorrect(msg, "echo -n '{0}' > %o0".format(msg))
 
+    @unittest.skip
```

Comment:
Just wanted to make sure you were aware of these "skip test" decorators in test_docker.py and test_simple.py

Reply:
The ngas server was down. Just re-enabled it now and will remove these skips